train.py

```diff
@@ -1,5 +1,8 @@
 import argparse
 import tensorflow as tf
+import multiprocessing
+import os
+from tensor2tensor.data_generators.text_encoder import PAD_ID, EOS_ID
```

Comment:
Maybe adding `tensor2tensor` as unconditioned dependency is too much? Especially if we just use the dataset generated by it?

Reply:
Another option is to copy/paste code from `tensor2tensor`, however, there is too much code  beyond  a simple call to `problem.dataset`, and what is now done by just 7 lines of code in `read_dataset_t2t_format`, may turn into several hundreds of lines. I've tried that )

Reply:
Of course, we can do it import in `if` blocks...

Reply:
Do you think creating conversion script is a bad idea (from t2t format)? This would have been isolated the dependency.

Copying is definitely not an option. :)

Reply:
The only benefit of that is features precalculation at conversion stage, which is very fast at T2T. However, we'll need to decode targets from numbers to characters again )